playground/mocks/data/idp/idx/authenticator-verification-select-authenticator-without-signed-nonce.json

```diff
@@ -0,0 +1,193 @@
+        "value": [
+          {
+            "name": "authenticator",
+            "type": "object",
+            "options": [
+              {
+                "label": "",
+                "value": {
+                  "form": {
+                    "value": [
+                      {
+                        "name": "id",
+                        "value": "auteq0lLiL9o1cYoN0g4",
+                        "required": true,
+                        "mutable": false
+                      },
+                      {
+                        "name": "methodType",
+                        "type": "string",
+                        "required": false,
+                        "mutable": true,
+                        "options": [
+                          {
+                            "value": "push",
+                            "label":  "Get a push notification"
+                          },
+                          {
+                            "value": "totp",
+                            "label" :  "Enter a code"
+                          }
+                        ]
+                      }
+                    ]
+                  }
+                },
+                "relatesTo": "$.authenticators.value[0]"
+              },
+              {
+                "label": "Okta Password",
+                "value": {
+                  "form": {
+                    "value": [
+                      {
+                        "name": "id",
+                        "value": "autepwdJFCg0J7vK60g4",
+                        "required": true,
+                        "mutable": false
+                      },
+                      {
+                        "name": "methodType",
+                        "value": "password",
+                        "required": false,
+                        "mutable": false
+                      }
+                    ]
+                  }
+                },
+                "relatesTo": "$.authenticatorEnrollments.value[2]"
+              }
+            ]
+          },
+          {
+            "name": "stateHandle",
+            "required": true,
+            "value": "02im-3M2f6UXHgNfS7Ns7C85EKHzGaKw0u1CC4p9_r",
+            "visible": false,
+            "mutable": false
+          }
+        ]
+      }
+    ]
+  },
+  "authenticators": {
+    "type": "array",
+    "value": [
+      {
+        "type": "app",
+        "id": "auteq0lLiL9o1cYoN0g4",
+        "methods": [
+          {
+            "type": "push"
+          },
+          {
+            "type": "totp"
+          },
+          {
+            "type": "signed_nonce"
+          }
+        ]
+      },
+      {
+        "type": "password",
+        "id": "autepwdJFCg0J7vK60g4",
+        "displayName": "Okta Password",
+        "methods": [
+          {
+            "type": "password"
+          }
+        ]
+      }
+    ]
+  },
+  "authenticatorEnrollments": {
+    "type": "array",
+    "value": [
+      {
+        "type": "app",
```

Comment:
@ManuMalhotra-okta  deviceKnown:true here if the device that the user is logging in from has a valid fastpass enrollment and we know the device id.

Reply:
@magizh-okta I thought we were putting it in the `authenticators` section. Both are ok from my side. Is it easier on your side if I add it to the `authenticatorEnrollments` section.

Reply:
My bad sorry it is in the authenticators section. I have the code accordingly I just left the comment in the wrong place .